tone-analyzer/src/main/java/com/ibm/watson/developer_cloud/tone_analyzer/v3/model/UtteranceAnalysis.java

```diff
@@ -0,0 +1,69 @@
+package com.ibm.watson.developer_cloud.tone_analyzer.v3.model;
+
+import com.google.gson.annotations.SerializedName;
+import com.ibm.watson.developer_cloud.service.model.GenericModel;
+
+import java.util.List;
+
+/**
+ * This object represents the results of Tone Chat analysis on an utterance. It has the utterance text from the original
+ * input. It holds a list of scores for individual chat tones.
+ */
+public class UtteranceAnalysis extends GenericModel {
+
+  @SerializedName("id")
+  int id;
```

Comment:
For the API, we have:

`int utterance_id;`
`String utterance_text;`

If I change the name in the java-sdk only, then I don't get correct response. Making a change to API might take time. If I don't, then the `checkstyle` step fails for the java-sdk.